pkg/models/public_dashboards.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+var (
+	ErrPublicDashboardFailedGenerateUniqueUid = DashboardErr{
+		Reason:     "Failed to generate unique dashboard id",
+		StatusCode: 500,
+	}
+)
+
+type PublicDashboardConfig struct {
+	IsPublic        bool            `json:"isPublic"`
+	PublicDashboard PublicDashboard `json:"publicDashboard"`
+}
+
+type PublicDashboard struct {
+	Uid               string `json:"uid" xorm:"uid"`
+	DashboardUid      string `json:"dashboardUid" xorm:"dashboard_uid"`
+	OrgId             int64  `json:"orgId" xorm:"org_id"`
+	RefreshRate       int64  `json:"refreshRate" xorm:"refresh_rate"`
+	TemplateVariables string `json:"templateVariables" xorm:"template_variables"`
+	TimeVariables     string `json:"timeVariables" xorm:"time_variables"`
```

Comment:
What does RefreshRate mean? 

splitting the settings up into two specific json blob columns seems the worst option, when we need to add more options we will need to add another column. A single Settings Json column would serve the same purpose on be more open to extension?

Reply:
`RefreshRate` was originally going to be used for caching, however, this has been removed for v1 along with `TemplateVariables`

While we're inching toward a v1 (no template variables), we want to be as explicit as possible about the data that we are storing until we have a dashboard schema that we can validate against. A migration to change this column will not be a big deal in the future should we need it.